docs/src/components/Footer.tsx

```diff
@@ -84,11 +80,7 @@ function Dot() {
   return <Desktop>·</Desktop>;
 }
 
-interface Props {
-  hasGradient?: boolean;
-}
-
-export default function Footer({ hasGradient }: Props) {
```

Comment:
Do we want to get rid of passing `hasGradient` property to the two or three usages of Footer? 🙂

Reply:
I decided to go a different way and simply always add a gradient in DocLayout, which is the goal of this PR anyway.